selenium-webdriver/remote.d.ts

```diff
@@ -66,6 +66,99 @@ declare namespace remote {
          */
         stop(): webdriver.promise.Promise<any>;
     }
+
+    module DriverService {
```

Comment:
Sometimes it's a fun challenge figuring out how to express perfectly valid JS constructs in a static type system.